EpiResponsivePicture/TypeScript/src/ResponsiveImage.tsx

```diff
@@ -0,0 +1,203 @@
+import * as React from "react";
+import { ReactNode } from "react";
+import { ResponsiveImageViewModel } from "./ResponsiveImageViewModel";
+import {
+  IAspectRatio,
+  ScaleMode,
+  IFocalPoint,
+  IPictureProfile,
+  IPictureSource,
+} from "../src/generated";
+export function getResizedImageUrl(
+  imageUrl: string,
+  width: number | null = null,
+  height: number | null = null,
+  mode: ScaleMode | null = null,
+  quality: number | null = null,
+  focalPoint: IFocalPoint | null = null
+) {
+  const urlParams = [];
+
+  if (width) {
+    urlParams.push(`width=${width.toString()}`);
```

Comment:
to be precise - on C# side we have an abstraction over generating url. Currently we're using ImageSharp as a resizing engine and it's using parameters like in this file - `height`, `rmode`, `rxy` etc. But in theory it's possible to switch engine to something different which will have `h`, `resize-mode` and `xy` instead and this will make this package to stop working properly. Should we address this issue?

Reply:
As these two packages are distributed from one repo, designed to work in combination and changed hand-in-hand I would leave it as is for now

Reply:
ok, in this case can we add a line of note in Readme that this npm package is meant to be working only with Imagesharp as a resizing engine - as in other cases resizing logic and parameters like height, width etc may differ?

Reply:
done